requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests>=0.34
+pil
```

Comment:
PIL is not a dependency of `kiwix-build` but of `kiwix-android-custom`. It is used in this script https://github.com/kiwix/kiwix-android-custom/blob/master/gen-custom-android-directory.py